Src/CSharpier.Tests/TestFiles/InitializerExpression/InitializerExpressions.cst

```diff
@@ -39,4 +39,13 @@ class ClassName
             longValue______________________________
         }
     };
+
+    void MethodName()
+    {
+        var x = new SomeClass
+        {
+            Property1 = true
+            // should indent
```

Comment:
Like... who would put a comment _there_?! :banana:

Reply:
Exactly. But I think I ran into this case in one of the repos I test format.